### ISSUE TITLE
RatingBuster.lua:3423: attempt to compare string with nil

### ISSUE THREAD

**Issue by Iowerth** [raethkcj/RatingBuster#142]
### Describe the bug

Hi. Version 1.7.5

```lua
2x RatingBuster/RatingBuster.lua:3423: attempt to compare string with nil
[string "@RatingBuster/RatingBuster.lua"]:3423: in function <RatingBuster/RatingBuster.lua:3422>
[string "=[C]"]: ?
[string "@RatingBuster/RatingBuster.lua"]:3810: in function `StatSummary'
[string "@RatingBuster/RatingBuster.lua"]:1830: in function `handler'
[string "@RatingBuster/libs/TipHooker-1.0-67029/TipHooker-1.0.lua"]:156: in function <...ns/RatingBuster/libs/TipHooker-1.0/TipHooker-1.0.lua:150>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: in function `SetItemByID'
[string "@AtlasLootClassic/Button/Item_type.lua"]:218: in function `OnEnter'
[string "@AtlasLootClassic/Button/Button.lua"]:125: in function <AtlasLootClassic/Button/Button.lua:118>

Locals:
a = <table> {
 1 = "Здоровье"
 2 = "1480.3 (|cffff1919-33.9|r)"
}
b = <table> {
 2 = "3.72% (|cff19ff19+3.72%|r)"
}
(*temporary) = "Здоровье"
(*temporary) = nil
(*temporary) = "attempt to compare string with nil"
```

### Addon Source

GitHub Releases

### Version

1.7.5

### Locale

Russian

### Talent Specialization

Blood Death Knight

### Race

Orc

**Comment by raethkcj** [comment]
Thank you for the report! I'm trying to reproduce the bug with my game in Russian, and all the of the Stat Summaries enabled, but so far I can't find an item that gives this error.

Are you getting this error when mousing over a particular item? Or maybe an item with a particular stat? The stat that's being converted to 3.72% seems to be the problem, but from the error I can't tell which stat it was supposed to be.

**Comment by Iowerth** [comment]
`(*temporary) = "Здоровье"` - it means HP.
I'm getting this error when hovering on `Conqueror's Darkruned Helmet (46115)` while having `Heroes' Scourgeborne Helmet (39619)` equipped.

**Comment by Iowerth** [comment]
I have found where is a problem.
If I turn off `Stat Summary` - `Stat - Physical` - `Sum Armor Penetration` then error disappears.

**Comment by raethkcj** [comment]
Thank you, should be fixed in 1.7.6!
